gltf/converter.py

```diff
@@ -586,7 +586,8 @@ def load_material(self, matid, gltf_mat):
             self.mat_mesh_map[matid] = []
 
         pmat = Material(matname)
-        pbr_fallback = {'index': '__pbr-fallback', 'texCoord': 0}
+        base_color_fallback = {'index': '__pbr-fallback', 'texCoord': 0}
+        metallic_roughness_fallback = {'index': '__pbr-fallback', 'texCoord': 0}
```

Comment:
Should they still both have the same index field?

Reply:
I thought about separating the two texture index, but nothing in glTF specification prevents two texture infos to reference the same texture (e.g. the occlusion texture could be packed with the metal-roughness texture).

On the other hand, the base color texture is using the sRGB color space and the metal-roughness the linear color space;  as the default value is all 1, in the end it does not change the result, but it starts to annoy me to have the wrong texture format applied...